addons/mail/static/src/models/message/message.js

```diff
@@ -256,6 +258,22 @@ function factory(dependencies) {
             }));
         }
 
+        /**
+         * Edits the message's body.
+         * 
+         * @param {String} body the new body of the message
+         */
+        async edit(body) {
+            const payload = await this.async(() => this.env.services.rpc({
+                route: '/mail/message/edit_body',
+                params: {
+                    id: this.id,
+                    body: body,
+                },
+            }));
+            this.messaging.models['mail.message'].insert(payload);
+        }
```

Comment:
I've renamed it to "updateContent" (since it will also be used to modify attachments), let me know if that name is clear enough or if you would like to choose a different name.

Reply:
It's better, thanks!